toolkit/tools/imagecustomizerapi/module.go

```diff
@@ -8,14 +8,23 @@ import (
 )
 
 type Module struct {
-	Name string `yaml:"Name"`
+	Name    string            `yaml:"Name"`
+	Options map[string]string `yaml:"Options,omitempty"`
 }
 
 func (m *Module) IsValid() error {
 	if m.Name == "" {
 		return fmt.Errorf("name of module may not be empty")
 	}
 
+	if m.Options != nil {
+		for key, value := range m.Options {
```

Comment:
Should be check that `key` and `value` don’t contain any spaces or newline characters?